src/Api/User.hs

```diff
@@ -73,7 +74,7 @@ waiMetrics = do
     increment "metrics"
     logDebugNS "web" "metrics"
     metr <- Metrics.getMetrics
-    liftIO $ mapM Counter.read =<< readIORef (metr ^. metricsCounters)
+    liftIO $ (fromList . toList) <$> (mapM Counter.read =<< readIORef (metr ^. metricsCounters))
```

Comment:
I'd change the return type to be `AppT m (HashMap Text Int64)` instead of doing the two conversions.